topics/dev/tutorials/tool-from-scratch/tutorial.md

```diff
@@ -0,0 +1,1182 @@
+>
+> > ### {% icon question %} Question
+> >
+> > What does your final file look like?
+> >
+> > > ### {% icon solution %} Solution
+> > >
+> > > {% raw %}
+> > > ```yaml
+> > > {% set name = "bellerophon" %}
+> > > {% set version = "1.0" %}
+> > > {% set sha256 = "036c5e23f53ed5b612525d3a32095acca073a9c8d2bf73883deb852c89f40dcf" %}
+> > >
+> > > package:
+> > >   name: {{ name|lower }}
+> > >   version: {{ version }}
+> > >
+> > > source:
+> > >   url: https://github.com/davebx/{{ name }}/archive/{{ version }}.tar.gz
+> > >   sha256: {{ sha256 }}
+> > >
+> > > build:
+> > >   noarch: python
+> > >   number: 0
+> > >   script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+> > >
+> > > requirements:
+> > >   host:
+> > >     - python
+> > >     - pip
+> > >   run:
+> > >     - python
+> > >     - pysam
+> > >
+> > > test:
+> > >   commands:
+> > >     - bellerophon --version
+> > >
+> > > about:
+> > >   home: https://github.com/davebx/bellerophon/
+> > >   license: MIT
+> > >   license_file: LICENSE
+> > >   summary: "Filter reads that span a mapping junction, retaining the 5'-side."
+> > >
+> > > ```
+> > > {% endraw %}
+> > {: .solution}
+> {: .question}
+{: .hands_on}
+
+## Creating a Pull Request
+
+After the recipe is complete, we can commit and push to our fork, so that the recipe can eventually be integrated into bioconda.
+
+
+> ### {% icon hands_on %} Hands-on: Creating the PR
+>
+> 1. We start by making sure we're on a branch, and that `recipes/bellerophon/meta.yaml` is the only modified file.
+>
+>    > ### {% icon code-in %} Input: Bash
+>    > ```bash
+>    > git status
+>    > ```
+>    {: .code-in}
+>
+>    > ### {% icon code-out %} Output
+>    >
+>    > ```
+>    > On branch bellerophon_bioconda
+>    > Untracked files:
+>    >   (use "git add <file>..." to include in what will be committed)
+>    >
+>    > 	recipes/bellerophon/
+>    >
+>    > no changes added to commit (use "git add" and/or "git commit -a")
+>    > ```
+>    {: .code-out}
+>
+> 2. Add your recipes folder, commit, and push it.
+>
+>    > ### {% icon code-in %} Input: Bash
+>    > ```bash
+>    > git add recipes/bellerophon
+>    > git commit -m 'Add recipe for bellerophon 1.0'
+>    > git push origin -u bellerophon_bioconda
+>    > ```
+>    {: .code-in}
+>
+>    Git will give you a url for creating the pull request, which can be clicked to be taken to github.
+>
+>    ![Git push](../../images/git_push.png "Expected output of the `git push` command")
+>
+> Once on the github pull request page, we can write a short description of the recipe we want to merge, and click the button.
+>
+> ![Github PR](../../images/create_pr.png "An open pull request for the bellerophon conda recipe")
+{: .hands_on}
+
+
+
+## Deploying a Conda Package
+
+Once our new recipe has been merged, installing the package should be fairly painless. Galaxy, if configured to do so,
```

Comment:
I think these steps should be contained within a hands-on box.

The hands on box usage was different across the tutorial, and it wasn't clear for me which sections I actually need to do.